src/shared/sol-iio.c

```diff
@@ -0,0 +1,990 @@
+#include <ctype.h>
```

Comment:
That's curious. I'm running it but it says nothing about this file. Does it still works? (Added license locally, btw.)